website/docs/r/rdb_user_beta.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_rdb_user_beta"
+description: |-
+  Manages Scaleway Database Users.
+---
+
+# scaleway_rdb_user_beta
+
+-> **Note:** This terraform resource is flagged beta and might include breaking change in future releases.
+
+Creates and manages Scaleway Database Users. For more information, see [the documentation](https://developers.scaleway.com/en/products/rdb/api).
+
+## Examples
+
+### Basic
+
+```hcl
+resource scaleway_rdb_user_beta db_admin {
+  	instance_id = scaleway_rdb_instance_beta.main.id
+  	name = "titi"
+  	password = "R34lP4sSw#Rd"
+  	is_admin = true
+}
+```
+
```

Comment:
can you add an example with the[ provider password](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password)?